systemd/bootc-publish-rhsm-facts.service

```diff
@@ -9,3 +9,4 @@ ExecStart=/usr/bin/bootc internals publish-rhsm-facts
 
 [Install]
 WantedBy=bootc-status-updated.target
+WantedBy=bootc-status-updated-onboot.target
```

Comment:
(Sorry again nonblocking but...I thought we'd make `bootc-status-onboot.target` have `Wants=bootc-status-updated.target`, then each individual unit is again just triggered solely by `bootc-status-updated.target)

Reply:
I think it has to be this way, because otherwise we'd have the same problem as before, except transitive.  `multi-user.target` would be active, so `bootc-status-updated-onboot.target` would also remain active, and in turn `bootc-status-updated.target` would remain active, so the path unit wouldn't retrigger things.

Reply:
(Queuing this PR to merge as is, so let's not update it further; this discussion is about followups)

What I was thinking was:

- `multi-user.target` -> `bootc-status-updated-onboot.target` -> `bootc-status-updated.target`
- `bootc-status-updated.path` -> `bootc-status-updated.target`

Now again the problem as is is:

> "If this option is set, a unit will be automatically cleaned up if no other active unit requires it. Defaults to false."

But isn't the fix there to also add `StopWhenUnneeded=true` to `bootc-status-updated-onboot.target`?

Reply:
No, because it's always "needed" by multi-user.target

Reply:
Really I think the RFE is probably for systemd to distinguish between "needed" for things that are soft (`Wants`) vs hard (`Requires`).  The docs specifically say:

```
If this option is set, a unit will be automatically cleaned up if no other active unit requires it.
```

But "requires" there is not `Requires`.